templates/web/base/report/_item.html

```diff
@@ -27,7 +27,7 @@
 [% END %]
 
 <li class="item-list__item item-list--reports__item [% item_extra_class %]"
-    data-report-id="[% problem.id | html %]" data-lastupdate="[% problem.lastupdate %]">
+    data-report-id="[% problem.id | html %]" data-lastupdate="[% problem.lastupdate %]" id="report-[% problem.id | html %]">
```

Comment:
As I've now changed the IDs to be unique in the list, I think it's safer to have the `data` element here too